src/Controls/src/Core/Handlers/Items2/iOS/LayoutFactory2.cs

```diff
@@ -83,8 +83,14 @@ static NSCollectionLayoutBoundarySupplementaryItem[] CreateSupplementaryItems(La
 		return [];
 	}
 
-	static UICollectionViewLayout CreateListLayout(UICollectionViewScrollDirection scrollDirection, LayoutGroupingInfo groupingInfo, LayoutHeaderFooterInfo layoutHeaderFooterInfo, LayoutSnapInfo snapInfo, NSCollectionLayoutDimension itemWidth, NSCollectionLayoutDimension itemHeight, NSCollectionLayoutDimension groupWidth, NSCollectionLayoutDimension groupHeight, double itemSpacing, Func<Thickness>? peekAreaInsetsFunc)
+	static UICollectionViewLayout CreateListLayout(UICollectionViewScrollDirection scrollDirection, LayoutGroupingInfo groupingInfo, LayoutHeaderFooterInfo layoutHeaderFooterInfo, LayoutSnapInfo snapInfo, NSCollectionLayoutDimension itemWidth, NSCollectionLayoutDimension itemHeight, NSCollectionLayoutDimension groupWidth, NSCollectionLayoutDimension groupHeight, double itemSpacing, Func<Thickness>? peekAreaInsetsFunc, ItemSizingStrategy itemSizingStrategy = ItemSizingStrategy.MeasureAllItems)
 	{
+		// For MeasureFirstItem, create a custom layout that measures the first item and applies that size to all items
+		if (itemSizingStrategy == ItemSizingStrategy.MeasureFirstItem)
+		{
+			return new MeasureFirstItemCollectionViewLayout(snapInfo, scrollDirection, itemSpacing, groupingInfo, layoutHeaderFooterInfo);
+		}
+
 		var layoutConfiguration = new UICollectionViewCompositionalLayoutConfiguration();
```

Comment:
Is there anyother way to make set the size of the item in UICollectionViewCompositionalLayoutConfiguration , without using UICollectionViewFlowLayout